src/platform/user/profile/vet360/containers/Vet360ProfileField.jsx

```diff
@@ -106,7 +106,13 @@ class Vet360ProfileField extends React.Component {
   };
 
   onSubmit = () => {
-    this.captureEvent('update-button');
+    // The validateAddress thunk will handle its own dataLayer additions
+    if (this.props.useAddressValidation) {
+      if (!this.props.fieldName.toLowerCase().includes('address')) {
```

Comment:
Should we set `fieldName` a default prop of an empty string so that this won't throw a runtime error in case it's not provided? 🙂